src/bootstrap-table.js

```diff
@@ -1894,6 +1894,10 @@
         this.$tableBody.find('tbody tr:first-child:not(.no-records-found) > td').each(function (i) {
             $footerTd.eq(i).outerWidth($(this).outerWidth());
         });
+        if (this.$tableFooter.find('table').width() != this.$tableBody.find('table').innerWidth())
```

Comment:
I think it's better to change it to:
```js
if (this.$tableFooter.find('table').width() != this.$tableBody.find('table:eq(0)').innerWidth()) {
    this.$tableFooter.find('table').width(this.$tableBody.find('table:eq(0)').innerWidth());
}
```
When the table body has sub table maybe cause error, what do you think so?